src/dataflow/components/nodes/dataflow-node.sass

```diff
@@ -172,6 +199,36 @@ $node-inner-width: 140px
         justify-content: space-evenly
         min-width: 151px
 
+        .output-container
+          margin: 0 3px 0 10px
+          padding: 3px
+          display: flex
+          justify-content: center
+          align-items: center
+          background-color: white
+          border-radius: 3px 12px 12px 3px
+          width: 140px
+          height: 30px
+          box-sizing: border-box
+          border: solid 2px white
+          &.sensor
+            background-color: $sensor-blue-readout
+          &.number
+            background-color: $sensor-blue-readout
```

Comment:
Is it correct that every class here has its own color except `number` which uses the `sensor` color? If so, would it make sense to make that explicit by combining them, e.g.:
```
 &.sensor, &.number
   background-color: $sensor-blue-readout
```

Reply:
I like it.  I don't anticipate those colors diverging at this point.